client/client/src/App.js

```diff
@@ -1,72 +0,0 @@
-import React, { Component } from 'react';
-import './App.css';
-import axios from 'axios'
-import AddAnimals from './components/AddAnimals'
-
-const BASE_URL = 'http://localhost:4000'
-
-class App extends Component {
-  constructor () {
-    super() 
-
-    this.state = {
-      animalsInZoo: {
-        name: '',
-        id: '',
-        imageUrl: ''
-      } ,
-      zoo: [],
-    }
-  }
-
-
-  componentDidMount(){
-    axios({
-      method: 'GET',
-      url: BASE_URL +'/api/zoo',
-    }).then(response => {
-      console.log('animals',response)
-      this.setState({ zoo: response.data })
-    })  
-  }
-
-  updateAnimals(animals) {
-    this.setState({ zoo: animals })
-  }
-
-  render() {
-    
-    
-    return (
-      
-      <div>
-       
-      <div style={{ margin: '0 auto', marginTop: 40 }}>
-        <AddAnimals updateAnimals={(animals) => this.setState({ zoo: animals })}/>
```

Comment:
this inline function is doing the same thing as the `updateAnimals` function you wrote above, however 2 things: 
* If you want `this.setState` to work in that function you have to define it using this syntax: 
```
updateAnimals = (animals) => {

}
```
if you don't, you'd have to bind in the constructor. This is maybe why you didn't end up using that function and instead did the inline.

Reply:
I do like how you used a component and passed a prop though, nice work 👍